telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/youtube/service/XmlParser.java

```diff
@@ -0,0 +1,11 @@
+package com.github.bumblebee.command.youtube.service;
+
+import javax.xml.bind.JAXBException;
+
+public interface XmlParser {
```

Comment:
I don't think that we need this interface. At lease with this name (too general).